inst/stan/wwinference.stan

```diff
@@ -187,9 +190,9 @@ transformed parameters {
   s = get_vl_trajectory(t_peak, viral_peak, dur_shed, gt_max);
 
   // Site level disease dynamic estimates!
-  i0_site_over_n = inv_logit(logit(i0_over_n) + eta_i0 * sigma_i0);
-  growth_site = initial_growth + eta_growth * sigma_growth; // site level growth rate
   for (i in 1:n_subpops) {
+    vector[n_weeks] log_r_site_t_in_weeks;
+    real log_i0_site = log(i_first_obs_over_n_site[i]) - uot * initial_exp_growth_rate_site[i];
```

Comment:
My suggestion around using `i0` to indicate the first observed time would of course require renaming this to `log_i_uot_site` or something that indicates its the log(I/N) at `t = - uot`

Reply:
We have used the t=0 as time of first observation convention in pyrenew. Would be nice to be consistent.

Reply:
@damonbayer that would suggest rewriting all the indexing in this model which doesn't seem worthwhile. I would suggest you have one approach per model and note down how to map between

Reply:
See discussion below. @kaitejohnson is interested in changing notation to make `t=0` correspond to the first observation, but if that is to happen it should be a separate issue+PR.

Reply:
And wouldn't change the actual time indexing, just how it is notated